framework/captcha/Captcha.php

```diff
@@ -164,18 +164,9 @@ protected function getClientOptions()
      */
     public static function checkRequirements()
     {
-        if (extension_loaded('imagick')) {
-            $imagickFormats = (new \Imagick())->queryFormats('PNG');
-            if (in_array('PNG', $imagickFormats, true)) {
-                return 'imagick';
-            }
-        }
-        if (extension_loaded('gd')) {
-            $gdInfo = gd_info();
-            if (!empty($gdInfo['FreeType Support'])) {
-                return 'gd';
-            }
-        }
-        throw new InvalidConfigException('Either GD PHP extension with FreeType support or ImageMagick PHP extension with PNG support is required.');
+        /* @var $imageModel ImageExtentionModel */
```

Comment:
@SilverFire This is not phpdoc, this is vardoc and this syntax is used multiple times in framework, because it has better support in editors.